src/reference/asciidoc/transformer.adoc

```diff
@@ -446,6 +446,61 @@ If the expression returns `null`, the `defaultType` is used.
 The `SimpleToAvroTransformer` also has a `setTypeExpression` method.
 This allows decoupling of the producer and consumer where the sender can set the header to some token representing the type and the consumer then maps that token to a type.
 
+[[Protobuf-transformers]]
+===== Protocol Buffers Transformers
+
+Version 6.1 adds support for transforming from and to https://protobuf.dev/[Protocol Buffers] data content.
+
+The `ToProtobufTransformer` transforms `com.google.protobuf.Message` message payloads into native byte array or json text payloads.
+The `application/x-protobuf` content type (default) produces byte array output payload.
+Set the `application/json` content type and add the `com.google.protobuf:protobuf-java-util` on the classpath to output text message payload. If the content type header is not set the `ToProtobufTransformer` defaults to `application/x-protobuf`.
```

Comment:
We prefer one line per sentence: https://asciidoctor.org/docs/asciidoc-recommended-practices/#one-sentence-per-line

Reply:
Doesn't look like you have resolved this and other concerns in docs.